apps/manuscripts/search_indexes.py

```diff
@@ -28,9 +33,37 @@ def prepare_image_availability(self, obj):
     def prepare_catalogue_numbers(self, obj):
         return obj.historical_item.get_catalogue_numbers_display()
 
+        # Extract and serialize the `Date` model fields
+
+    def prepare_date(self, obj):
+        if obj.historical_item and obj.historical_item.date:
```

Comment:
```suggestion
        if obj.historical_item.date:
```
Because we're sure there will always be a historical item (it's not nullable).